{
	"took": 3,
	"timed_out": false,
	"_shards": {
		"total": 4,
		"successful": 4,
		"failed": 0
	},
	"hits": {
		"total": 1005,
		"max_score": 1.0,
		"hits": [{
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "7bec8900-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 179495336,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:02:14",
				"version": "1.1",
				"timestamp": "2015-11-11 21:02:13.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "7bec8900-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(1440) TradeReportProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(1440) TradeReportProvider, ConnectionRefused: Connection refuse",
				"Category": "MI AIF",
				"SourceName": "AMS_HEL",
				"source": "sse131.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "7d21fbc0-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 179495346,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:02:16",
				"version": "1.1",
				"timestamp": "2015-11-11 21:02:16.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "7d21fbc0-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(1440) TradeReportProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(1440) TradeReportProvider, ConnectionRefused: Connection refuse",
				"Category": "MI AIF",
				"SourceName": "AMS_HEL",
				"source": "sse131.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "794bc672-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 9317938,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:02:09",
				"version": "1.1",
				"timestamp": "2015-11-11 21:02:09.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "794bc672-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "ServerGuard: AFG_JPT terminated - Restarting in 1 seconds...",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "ServerGuard: AFG_JPT terminated - Restarting in 1 seconds...",
				"Category": "ServerGuard",
				"SourceName": "AFG_JPT",
				"source": "sse158.carnnet.com",
				"Task": 8,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "b8f75dc3-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 179495621,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:03:57",
				"version": "1.1",
				"timestamp": "2015-11-11 21:03:56.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "b8f75dc3-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(1424) OrderProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(1424) OrderProvider, ConnectionRefused: Connection refused from",
				"Category": "MI AIF",
				"SourceName": "AMS_HEL",
				"source": "sse131.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "b067bf13-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 102851253,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:03:42",
				"version": "1.1",
				"timestamp": "2015-11-11 21:03:42.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "b067bf13-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(D30) OrderProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(D30) OrderProvider, ConnectionRefused: Connection refused from ",
				"Category": "MI AIF",
				"SourceName": "AMS_INE",
				"source": "sse111.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "b0680d33-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 102851257,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:03:42",
				"version": "1.1",
				"timestamp": "2015-11-11 21:03:42.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "b0680d33-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(3A4) UnsolicitedProvider1, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(3A4) UnsolicitedProvider1, ConnectionRefused: Connection refuse",
				"Category": "MI AIF",
				"SourceName": "AMS_INE",
				"source": "sse111.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "b19d58e2-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 102851268,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:03:44",
				"version": "1.1",
				"timestamp": "2015-11-11 21:03:44.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "b19d58e2-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(D30) OrderProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(D30) OrderProvider, ConnectionRefused: Connection refused from ",
				"Category": "MI AIF",
				"SourceName": "AMS_INE",
				"source": "sse111.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "b19df521-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 102851272,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:03:44",
				"version": "1.1",
				"timestamp": "2015-11-11 21:03:44.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "b19df521-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(3A4) UnsolicitedProvider1, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(3A4) UnsolicitedProvider1, ConnectionRefused: Connection refuse",
				"Category": "MI AIF",
				"SourceName": "AMS_INE",
				"source": "sse111.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "ff3f2832-88b7-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 20894148,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:05:54",
				"version": "1.1",
				"timestamp": "2015-11-11 21:05:54.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "ff3f2832-88b7-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(1D00) OrderProvider, ConnectionRefused: Connection refused from other side : \nError code : 0\nError text : IOException: connect () failed: No connection could be made because the target machine actively refused it.",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(1D00) OrderProvider, ConnectionRefused: Connection refused from",
				"Category": "MI AIF",
				"SourceName": "AMS_HEL",
				"source": "sse190.carnnet.com",
				"Task": 839,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}, {
			"_index": "graylog2_21",
			"_type": "message",
			"_id": "04934af2-88b8-11e5-bd0b-005056a2068a",
			"_score": 1.0,
			"_source": {
				"RecordNumber": 9320803,
				"Channel": "Application",
				"gl2_source_node": "01ba828f-3818-4dac-8765-f54b4934d58a",
				"EventReceivedTime": "2015-11-11 22:06:03",
				"version": "1.1",
				"timestamp": "2015-11-11 21:06:02.000",
				"SourceModuleName": "in",
				"Severity": "ERROR",
				"level": 3,
				"_id": "04934af2-88b8-11e5-bd0b-005056a2068a",
				"SeverityValue": 4,
				"gl2_source_input": "560fc23ee4b0ff480ccf0271",
				"Keywords": 36028797018963968,
				"SourceModuleType": "im_msvistalog",
				"full_message": "(8FC) FixProvider, Sending logout with text :: Low sequence number detected. Expected : 1199. Received : 13",
				"ThreadID": 0,
				"EventID": 4,
				"EventType": "ERROR",
				"ProcessID": 0,
				"message": "(8FC) FixProvider, Sending logout with text :: Low sequence numb",
				"Category": "AIF",
				"SourceName": "AFG_JPT",
				"source": "sse158.carnnet.com",
				"Task": 850,
				"streams": ["56137f99e4b0d01f53360fa2", "5612525be4b02a8fb52478a4"]
			}
		}]
	}
}